contracts/bulkers/BaseBulker.sol

```diff
@@ -1,33 +1,34 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity 0.8.15;
 
-import "./CometInterface.sol";
-import "./ERC20.sol";
-import "./IWETH9.sol";
+import "../CometInterface.sol";
+import "../ERC20.sol";
+import "../IWETH9.sol";
 
 interface IClaimable {
     function claim(address comet, address src, bool shouldAccrue) external;
 
     function claimTo(address comet, address src, address to, bool shouldAccrue) external;
 }
 
-contract Bulker {
+contract BaseBulker {
```

Comment:
Can't comment on line 35 so going to comment here instead. Should we go ahead and make the base even more generic by renaming `weth` to `wrappedNativeToken`?

Reply:
Yeah, it's a good question. `wrappedNativeToken` will be the more accurate name once we're using the bulker for wrapped Matic and wrapped Avax.

The downside is that it will break the existing interface. With these changes, the existing, deployed Bulkers and the new MainnetBulker have the same interface (the MainnetBulker just adds a couple additional actions).

But if we update the constants to be `ACTION_SUPPLY_NATIVE_TOKEN` instead of `ACTION_SUPPLY_ETH`, then we'll need to either wrap deployment-specific scenarios or replace the deployed bulkers.

Reply:
IMO it's fine to temporarily have both interfaces in the repo so scenarios can still use the old Bulker interface until we complete the migration on-chain. The cost of not making this change is that we'll either need a separate `BaseBulker` for the other chains or the bulker on other chains will incorrectly refer to `WMATIC` or `WAVAX` as `WETH`.

Reply:
sounds good. updated all references to `eth`/`weth` to be `nativeToken`/`wrappedNativeToken`